### ISSUE TITLE
telnetlib deprecated in python 3.13

### ISSUE THREAD

**Issue by johanreinalda** [Juniper/py-junos-eznc#1324]
Just FYI: 
using junos-eznc with python3.11, I get this (when warnings enabled):

.../venv/lib/python3.11/site-packages/jnpr/junos/transport/tty_telnet.py:2: DeprecationWarning: 'telnetlib' is deprecated and slated for removal in Python 3.13

3.13 will be release on Oct 1, 2024...

**Comment by dineshbaburam91** [comment]
@johanreinalda 
PyEZ supports Python 3.12 and will be validated using official Python 3.13. The module will be removed if it is deprecated in Python 3.13.

**Comment by ktbyers** [comment]
I vendored telnetlib in Netmiko (should be there in Netmiko 4.4.0 and later):

```python
from netmiko._telnetlib import telnetlib
```

I guess the other solutions are to drop telnet support or vendor it yourself (or find a different released version of telnetlib that you trust).

telnetlib is deprecated in the sense that it is no longer built into Python itself (as of PY3.13).

**Comment by ktbyers** [comment]
@dineshbaburam91 What is the Python 3.13 plan for PyEZ?

PY3.13 has now been released for several months.

I don't much care one way or another if PyEZ supports telnetlib, but it needs to work with PY3.13. This is causing issues for me with respect to both Napalm and Nornir libraries.

**Comment by ktbyers** [comment]
Created this PR here:

https://github.com/Juniper/py-junos-eznc/pull/1352

Open to other solutions, but definitely needs a fix.

**Comment by ktbyers** [comment]
Any update on this issue (the lack of PY3.13 support for PyEZ)?

**Comment by johanreinalda** [comment]
Yes, I would also like to see some movement here. I've reached out to our Juniper SE to see what he can shake loose internally...

**Comment by chidanandpujar** [comment]
Hi @ktbyers @johanreinalda 
   @dineshbaburam91  has fixed this issue and merged the changes via https://github.com/Juniper/py-junos-eznc/pull/1358

Thanks
Chidanand

**Comment by ktbyers** [comment]
Thanks @chidanandpujar and @dineshbaburam91
